index.html

```diff
@@ -61,6 +61,172 @@ <h2 class="body-text-header">I'm a Software Developer</h2>
 </svg>
 </div>
       </section>
+      <section class="project">
+        <div class="project-content">
+          <h3 class="underlined">My Recent Works</h3>
+        <div class="project-list">
+          <div class="project-item">
+            <div class="project-item-img">
+          <img src="/img/tonic2.png" alt="" class="img">
+        </div>
+      </div>
+      <div class="project-disc">
+      <div class="project-heading">Multi-Post Stories</div>
```

Comment:
- [ ] If you notice, you discover that this line is a `heading` title describing the `p` that follows. Kindly use a `hx` tag e.g. `h2`, `h3`...`h6` to make your code more semantic just like you did with other headers. It also helps screen readers to correctly present the content of your page. :wink: